include/libopencm3/nrf5/common/pwm_common.h

```diff
@@ -0,0 +1,527 @@
+/** @addtogroup timer_defines
```

Comment:
wrong group again, please check how the generated doxygen looks and check the rest of these yourself.